app/views/simple_calendar/_month_calendar.html.haml

```diff
@@ -1,14 +1,19 @@
 .simple-calendar
   .calendar-heading
-    = link_to t('simple_calendar.previous', default: 'Previous'), calendar.url_for_previous_view
+    %span.calendar-title
+    = @calendar.name
+
+  - @inactive_calendars.each do |calendar|
+    = link_to calendar.name, calendar_path(calendar), class: "btn btn-default"
+
+  .calendar-navigation
+    = link_to t('simple_calendar.previous', default: '<<'), calendar.url_for_previous_view, class: "btn btn-md"
```

Comment:
where does `url_for_previous_view` and `url_for_next_view` come from?

It also seems like you're doing navigation in two different ways - one way is to loop through all the inactive_calendars & display their links, the other is to use this previous calendar/next calendar view. I don't think we need both.

Reply:
ah wait, I just pulled this & tested it out - these arrows are for going from month to month! This is super awesome! It's a bit unclear from the code here tho I think... Is there any way to change some of the naming here to make it clearer? However, I fear this functionality might just come from the gem. :thinking:

Reply:
Yes it comes from the gem :woman_shrugging: